spec/css/media/whitespace.hrx

```diff
@@ -18,6 +18,42 @@ Error: Expected identifier.
   '
   input.sass 1:7  root stylesheet
 
+<===>
+================================================================================
+<===> error/logic_sequence/before_operator/sass/input.sass
```

Comment:
The error directory should go at the end of the HRX file.

Reply:
Moved in cd4eced4e5e1125de252490046e5eff6d498da20